README.md

```diff
@@ -153,6 +153,16 @@ Follow the steps in [Procfile.learner](./Procfile.learner) to add a learner node
 goreman -f ./Procfile.learner start
 ```
 
+### Install etcd client v3
+
+```bash
+go get go.etcd.io/etcd/client/v3
+```
+
+`etcd/clientv3` is the official Go etcd client for v3. Read more:[etcd client v3 install](https://github.com/etcd-io/etcd/tree/main/client/v3#install)
+
+Notes to resolve dependency and compatibility issues for go modules: [#13108](https://github.com/etcd-io/etcd/issues/13108)
```

Comment:
Thanks @forging2012 One comment I have is we shouldn't add Issue as a pointer in the ReadMe. As @lilic suggested in the issue, if there is anything missing in the doc then we should add instructions there. Issue discussion can be added in the future and can sometime diverse from original discussion. Thanks!

Reply:
Understood. I have modified it to your meaning so that it is straightforward and does not cause misunderstanding and introduce redundant original discussion.